app/templating/metadata_template_preprocessor.py

```diff
@@ -55,6 +55,7 @@ def _build_survey_meta(self, questionnaire_schema):
             "start_date":  self._format_date(metadata["ref_p_start_date"]),
             "end_date": self._format_date(metadata["ref_p_end_date"]),
             "employment_date": self._format_date(metadata["employment_date"]),
+            "region_code": metadata["region_code"] if 'region_code' in metadata else None,
```

Comment:
Would this not fail with KeyError if region_code wasn't in metadata or is that a misunderstanding?